rocAL/include/loaders/circular_buffer.h

```diff
@@ -31,8 +31,8 @@ THE SOFTWARE.
 #include "commons.h"
 #include "device_manager.h"
 #include "device_manager_hip.h"
-struct decoded_image_info {
-    std::vector<std::string> _image_names;
+struct decoded_sample_info {
```

Comment:
"data" is more appropriate name than sample

Reply:
Changed to DecodedDataInfo